runner/migrations/0013_merge_20200117_1842.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.8 on 2020-01-17 18:42
```

Comment:
Do we need this migration?

Reply:
it was auto-generated so I assumed its needed to migrate

Reply:
Yes, I see that. Can you just check do we need it? Because there is no operation in the migration, so if we don't need it I prefer removing it.